scripts/convertBooks.ts

```diff
@@ -32,11 +32,48 @@ function removeStrongNumberReferences(text: string, _bcId: string, _bookId: stri
     return text.replace(/(\\\+?w) ([^|]*)\|strong="[^"]*"\1\*/g, '$2');
 }
 
+function removeMissingFigures(text: string, _bcId: string, _bookId: string): string {
+    // Regular expression to match \fig markers
+    const figRegex = /\\fig\s(.*?)\\fig\*/g;
+
+    // Replace each \fig marker with the appropriate action
+    return text.replace(figRegex, (match, figContent) => {
+        // Split the content of the \fig marker by pipe (|)
+        const parts = figContent.split('|');
+
+        // Extract the image source from the first part
+        let imageSource;
+        if (parts[1].startsWith('src="')) {
```

Comment:
Does it have to start with this string, or should this string just be included somewhere if it is the new format?

Reply:
Hmm ... I guess it technically doesn't have to start with it.